pkg/internal/kube/cache_svc_client.go

```diff
@@ -53,6 +56,15 @@ func (sc *cacheSvcClient) Start(ctx context.Context) {
 			}
 		}
 	}()
+	sc.log.Debug("waiting for K8s metadata synchronization")
```

Comment:
I think we may want to turn this into an Info message, so we can tell what's happening if Beyla is simply stuck and not continuing. We can put the current timeout value here and tell people if they need to configure it. WDYT?